gamestonk_terminal/jupyter/dashboards/correlation.ipynb

```diff
@@ -0,0 +1,199 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
```

Comment:
shall we clear the notebook outputs and metadata by default? i can try to add it to the pre-commit hook

Reply:
That would be very helpful. I already forgot that little line of code you sent me, but having it automatically run would be tremendous.

Reply:
@piiq I added a pre-commit hook to remove kernel metadata.